web/server/codechecker_server/api/report_server.py

```diff
@@ -1397,12 +1455,27 @@ def getComments(self, report_id):
                     .order_by(Comment.created_at.desc()) \
                     .all()
 
+                context = webserver_context.get_context()
                 for comment in comments:
+                    message = comment.message
+
+                    sys_comment = comment_kind_str(ttypes.CommentKind.SYSTEM)
+                    if comment.kind == sys_comment:
+                        elements = shlex.split(message)
```

Comment:
why do we need the shlex.split here?

Reply:
Before we store a system comment in the database, we will escape the white spaces (`escape_whitespaces`). We are using `shlex` here to split this escaped string properly.